devel/unicorn/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        unicorn-engine unicorn 2.0.0-rc7
```

Comment:
please package the stable release here, if you want it's possible to add a `-devel` port that tracks a development snapshot

Reply:
the stable release is broken on Apple Silicon Macs. 2.0 has not been released as stable yet. Should I just push a -devel Portfile?

Reply:
that's kind of unconventional as well to only add a `-devel` port. Any insights on how long an actual 2.x release going to take; if it's expected shortly perhaps just wait for that and then add the port